### DISCUSSION TITLE
Removal of `Language(path, name)`, next steps

### DISCUSSION THREAD

**Discussion by ToBeReplaced** [tree-sitter/py-tree-sitter#244] [General]
I use this library to load and work with tree-sitter libraries of custom languages. As I understand it, with the new release, the only way to load a binding for a custom language is to adopt a build process like the one found in `tree-sitter-python` and make those bindings available to this library's environment. This will not be possible in my case, and in general would not work with reloads, so I will not be able to update this library.

I can pin the old version, but I like some of the recent updates (e.g. better type hinting).

What do you think is the best way forward? I am wondering if the ability to create a `Language` from `(path, name)` might be viable as a separate library, if you think it doesn't belong in this library directly. What would be your recommendation?

**Comment by leotrs** [comment]
Where could I get more context about this? This would also mean a breaking change in my application.

**Comment by ObserverOfTime** [comment]
Check https://github.com/grantjenks/py-tree-sitter-languages/pull/65. You can fork and adapt it for the languages you need.
